tests/buildtest/crates/invalid-cargotoml-content-type-in-deps/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "invalid-cargotoml-content-type-in-deps"
+version = "0.1.0"
+authors = []
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+libressl-pnacl-sys = "=2.1.0"

```

Comment:
can you explain where this is wrong?

Reply:
If you look at the Cargo.tomlin the crates file <https://static.crates.io/crates/libressl-pnacl-sys/libressl-pnacl-sys-2.1.0.crate>, there the `[lib]` table is declared as `[[lib]]` an array of tables, which is invalid and rejected by cargo

Reply:
as for the name: in a **dep**endency the **type** of a value in the **Cargo.toml** is **invalid**